test/server.js

```diff
@@ -0,0 +1,250 @@
+const assert = require('assert').strict;
+const { AssertionError } = require('assert');
+const expect = require('expect');
+const app = require('express')();
+const server = require('http').Server(app);
+const io = require('socket.io')(server);
+const toJsonSchema = require('@openapi-contrib/openapi-schema-to-json-schema');
+const Ajv = require('ajv');
+const bodyParser = require('body-parser');
+const { isValidUUIDV4 } = require('is-valid-uuid-v4');
+
+const metricsApi = require('./openapi.json');
+const harFixtures = require('./payloads');
+
+const port = 3000;
```

Comment:
nit: we may want to pick a different port given 3000 may clash if we have the readme app running at the same time